packages/airnode-examples/integrations/relay-security-schemes/config.example.json

```diff
@@ -0,0 +1,135 @@
+{
+  "chains": [
+    {
+      "maxConcurrency": 100,
+      "authorizers": [],
+      "contracts": {
+        "AirnodeRrp": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
+      },
+      "id": "31337",
+      "providers": {
+        "exampleProvider": {
+          "url": "${PROVIDER_URL}"
+        }
+      },
+      "type": "evm",
+      "options": {
+        "txType": "eip1559",
+        "baseFeeMultiplier": "2",
+        "priorityFee": {
+          "value": "3.12",
+          "unit": "gwei"
+        }
+      }
+    }
+  ],
+  "nodeSettings": {
+    "cloudProvider": {
+      "type": "local"
+    },
+    "airnodeWalletMnemonic": "${AIRNODE_WALLET_MNEMONIC}",
+    "heartbeat": {
+      "enabled": false
+    },
+    "httpGateway": {
+      "enabled": false
+    },
+    "logFormat": "plain",
+    "logLevel": "INFO",
+    "nodeVersion": "0.4.0",
+    "stage": "dev",
+    "skipValidation": true
+  },
+  "triggers": {
+    "rrp": [
+      {
+        "endpointId": "0x825a9b8e4e78772cd07cc4307de6737b67cf6d624fff2fa56f71318e479f624b",
+        "oisTitle": "Relay Security Schemes via httpbin",
+        "endpointName": "httpbinRelaySecuritySchemes"
+      }
+    ]
+  },
+  "ois": [
+    {
+      "oisFormat": "1.0.0",
+      "title": "Relay Security Schemes via httpbin",
+      "version": "1.0.0",
+      "apiSpecifications": {
+        "servers": [
+          {
+            "url": "https://httpbin.org"
+          }
+        ],
+        "paths": {
+          "/get": {
+            "get": {
+              "parameters": []
+            }
+          }
+        },
+        "components": {
+          "securitySchemes": {
+            "relayRequesterAddress": {
+              "in": "header",
+              "type": "relayRequesterAddress",
+              "name": "requesterAddress"
+            },
+            "relaySponsorAddress": {
+              "in": "query",
+              "type": "relaySponsorAddress",
+              "name": "sponsorAddress"
+            },
+            "relaySponsorWalletAddress": {
+              "in": "query",
+              "type": "relaySponsorWalletAddress",
+              "name": "sponsorWalletAddress"
+            },
+            "relayChainId": {
+              "in": "query",
```

Comment:
Nit: Consider using cookie? (Not sure if the mock API returns that as well)

Reply:
I can if there is a way to parse `31337` out of this httpbin.org JSON response. My question is what should `_path` be? For example `headers.Cookie.???`? The challenge is the `_type` for `chainId` that I have is `uint256`. I suppose I could switch it to `string32` and use just `headers.Cookie`, but then you have to decode and split the text on `=` and `;` which feels like a hack.
```json
"headers": {
  "Accept":"application/json, text/plain, */*",
  "Cookie":"chainId=31337;",
  "Host":"httpbin.org",
```

Reply:
Actually since `chainType` is `string32` already anyway, I'll use cookie there.

Reply:
Ok I implemented cookie for `relayChainType`. I've updated the README to mention values being passed by `query`, `header`, and `cookie`.